File: app/src/main/java/com/ydws/game/activity/PropActivity.java
package com.ydws.game.activity;

import android.content.Intent;
import android.view.View;

import com.ydws.game.R;
import com.ydws.game.base.BaseAbstractActivity;

/**
 * 道具回收
 */
public class PropActivity extends BaseAbstractActivity implements View.OnClickListener {
    @Override
    public int getContentLayoutID() {
        return R.layout.activity_prop;
    }

    @Override
    public void initViews() {
findViewById(R.id.submit).setOnClickListener(this);
    }

    @Override
    public void initData() {

    }

    @Override
    public void onClick(View view) {
        switch (view.getId()){
            case R.id.submit:
                break;
        }
    }
}

File: app/src/main/java/com/ydws/game/activity/PropActivity.kt
package com.ydws.game.activity

import android.content.Intent
import android.databinding.DataBindingUtil
import android.os.Bundle
import android.view.View

import com.ydws.game.R
import com.ydws.game.base.BaseAbstractActivity
import com.ydws.game.databinding.ActivityPropBinding

/**
 * 道具回收
 */
class PropActivity : BaseAbstractActivity(), View.OnClickListener {
    private lateinit var activityPropBinding: ActivityPropBinding;
    override fun getContentLayoutID(): Int {
        return R.layout.activity_prop
    }


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityPropBinding = DataBindingUtil.setContentView(this,R.layout.activity_prop)

    }
    override fun initViews() {
        findViewById<View>(R.id.submit).setOnClickListener(this)
    }

    override fun initData() {

    }

    override fun onClick(view: View) {
        when (view.id) {
            R.id.submit -> {
            }
        }
    }
}
